homeassistant/components/ovos/config_flow.py

```diff
@@ -0,0 +1,78 @@
+"""Config flow for OVOS integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN
+from .notify import OvosNotificationService
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required("host"): str,
+        vol.Optional("ovos_port", default=8181): int,
+    }
+)
+
+
+async def validate_input(hass: HomeAssistant, data: dict[str, Any]) -> dict[str, Any]:
+    """Validate the user input allows us to connect.
+
+    Data has the keys from STEP_USER_DATA_SCHEMA with values provided by the user.
+    """
+    try:
+        hub: Any = await hass.async_add_executor_job(
+            OvosNotificationService(data["host"], data["ovos_port"]).authenticate()
```

Comment:
You need to pass the function to the executor job. 
```suggestion
            OvosNotificationService(data["host"], data["ovos_port"]).authenticate
```